protocol/communities/manager_archive_mobile.go

```diff
@@ -0,0 +1,46 @@
+//go:build disable_torrent
+// +build disable_torrent
+
+package communities
+
+import (
+	"crypto/ecdsa"
+	"time"
+
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+type ArchiveManagerMobile struct{}
```

Comment:
I don't think the _name of the struct_ should represent _how we use it_, but _it's implementation_.
If at some point we decide that mobile should have torrents and desktop shouldn't, would be weird to rename the struct.

I recently found `zap.NewNop`, which creates a no-op logger. I liked the naming of it 🙂 

```suggestion
type ArchiveManagerNop struct{}
```

Reply:
That is a better name I'll do this.

Reply:
https://github.com/status-im/status-go/pull/5257/commits/048172d5b4249eb5f78d094a9362651ef1904340